### PR TITLE
Add transactionTypes to DB Schema for CIBA

### PR DESCRIPTION
**Pull request by moisesolmedo-okta**
### Problem Analysis (Technical)
for CIBA we need to know what type of transactions an enrollment supports and persist it across sessions.

### Solution (Technical)
We need to persist transactionTypes by enrollment. Hence, adding `transactionTypes` column to `EnrolledMethod` table.

### Steps to reproduce:

Actual result:

Expected result:

### CODE REVIEW

File: Sources/DeviceAuthenticator/Storage/SQLite/schemas/SQLiteSchema_200.swift [hunk 1]
```diff
@@ -43,7 +43,8 @@ class SQLiteSchema_200: SQLiteSchemaProtocol {
     'algorithm' INTEGER DEFAULT NULL,
     'sharedSecret' BLOB DEFAULT NULL,
     'createdTimestamp' TEXT DEFAULT NULL,
-    'updatedTimestamp' TEXT DEFAULT NULL
+    'updatedTimestamp' TEXT DEFAULT NULL,
+    'transactionTypes' INTEGER DEFAULT NULL
```
Comment (IldarAbdullin-okta) [inline Sources/DeviceAuthenticator/Storage/SQLite/schemas/SQLiteSchema_200.swift:47]: Could you put it after `sharedSecret`?

Comment (IldarAbdullin-okta) [inline Sources/DeviceAuthenticator/Storage/SQLite/schemas/SQLiteSchema_200.swift:47]:
```suggestion
    'transactionTypes' INTEGER DEFAULT 1
```

**Review by IldarAbdullin-okta — COMMENTED** [review]
Looks good. Please review comments

File: Sources/DeviceAuthenticator/Storage/SQLite/schemas/SQLiteSchema_200.swift (hunk 1 above)
Comment (FeiChen-okta) [inline Sources/DeviceAuthenticator/Storage/SQLite/schemas/SQLiteSchema_200.swift:47]: transactionTypes is a enum?

Comment (moisesolmedo-okta) [reply Sources/DeviceAuthenticator/Storage/SQLite/schemas/SQLiteSchema_200.swift:47]: Yes. It was @IldarAbdullin-okta suggestion so we can treat them as bitmask enums in the codebase. 
1 - (`1 << 0`) -> login
3 - (`1 << 0`) -> login and ciba

Comment (moisesolmedo-okta) [reply Sources/DeviceAuthenticator/Storage/SQLite/schemas/SQLiteSchema_200.swift:47]: what's the difference?

Comment (moisesolmedo-okta) [reply Sources/DeviceAuthenticator/Storage/SQLite/schemas/SQLiteSchema_200.swift]: Ok. I was planning to treat a NULL value as `login` type in code but that works too.

Comment (IldarAbdullin-okta) [reply Sources/DeviceAuthenticator/Storage/SQLite/schemas/SQLiteSchema_200.swift]: Yeah, I think bitmask perfectly fits for encoding capabilities and it is scalable

**Review by IldarAbdullin-okta — APPROVED** [review]
LGTM!

**Pull request merged**